### PR TITLE
Owntracks 1.9 binding information added

### PR DESCRIPTION
**Pull request by smar000**

### CODE REVIEW

File: _data/oh1addons_infos.yml [hunk 1]
```diff
@@ -161,6 +161,10 @@
   description: This binding allows openHAB to act as an MQTT client, so that openHAB items can send and receive MQTT messages to/from an MQTT broker
   wiki_url: https://github.com/openhab/openhab/wiki/MQTT-Binding
 
+- label: Owntracks Binding
+  description: Owntracks (formerly known as Mqttitude) provides device location data via an mqtt bridge to openHAB
```
Comment (watou) [inline _data/oh1addons_infos.yml]: It sort of doesn't matter, but it was called MQTTitude before OwnTracks (note differences in capitalization).  Also mqtt would be better as MQTT.

File: _data/oh1addons.csv [hunk 2]
```diff
@@ -57,6 +57,7 @@ binding,Nikobus Binding
 binding,Novelan Heatpump Binding
 binding,OpenEnergyMonitor Binding
 binding,OneWire Binding
+binding,Owntracks (mqttitude) Binding
```
Comment (smar000) [inline _data/oh1addons.csv]: Ok. Can I leave this as is for now or do I need to do anything else?

File: _data/oh1addons_infos.yml (hunk 1 above)
Comment (smar000) [inline _data/oh1addons_infos.yml]: Done.

File: _data/oh1addons.csv (hunk 2 above)
Comment (kaikreuzer) [inline _data/oh1addons.csv]: you can leave it, it will be automatically overwritten once I update the generated content.

**Pull request merged**